rma/tests/test_supplier_rma.py

```diff
@@ -100,58 +96,52 @@ def test_supplier_rma(self):
                           "Incorrect number of pickings created")
-        picking_out = pickings[1]
+        picking_out = pickings[0]
         moves = picking_out.move_lines
-        self.assertEquals(len(moves), 3,
+        self.assertEquals(len(moves), 2,
                           "Incorrect number of moves created")
         for line in self.rma_supplier_id.rma_line_ids:
-            self.assertEquals(line.qty_to_deliver, 0,
-                              "Wrong qty to deliver")
-            self.assertEquals(line.qty_outgoing, 0,
-                              "Wrong qty outgoing")
+            self.assertEquals(line.qty_incoming, 0,
+                              "Wrong qty incoming")
             self.assertEquals(line.qty_received, 0,
                               "Wrong qty received")
             if line.product_id == self.product_1:
                 self.assertEquals(line.qty_to_receive, 3,
                                   "Wrong qty to receive")
-                self.assertEquals(line.qty_incoming, 3,
+                self.assertEquals(line.qty_incoming, 0,
                                   "Wrong qty incoming")
                 self.assertEquals(line.qty_delivered, 3,
                                   "Wrong qty delivered")
             if line.product_id == self.product_2:
                 self.assertEquals(line.qty_to_receive, 5,
                                   "Wrong qty to receive")
-                self.assertEquals(line.qty_incoming, 5,
-                                  "Wrong qty incoming")
-                self.assertEquals(line.qty_delivered, 5,
-                                  "Wrong qty delivered")
+                self.assertEquals(line.qty_to_deliver, 5,
+                                  "Wrong qty to deliver")
             if line.product_id == self.product_3:
                 self.assertEquals(line.qty_to_receive, 2,
                                   "Wrong qty to receive")
-                self.assertEquals(line.qty_incoming, 2,
-                                  "Wrong qty incoming")
-                self.assertEquals(line.qty_delivered, 2,
-                                  "Wrong qty delivered")
+                self.assertEquals(line.qty_to_deliver, 2,
+                                  "Wrong qty to deliver")
         picking_out.action_assign()
         picking_out.do_transfer()
-        for line in self.rma_supplier_id.rma_line_ids:
-            self.assertEquals(line.qty_to_receive, 0,
+        for line in self.rma_supplier_id.rma_line_ids[0]:
+            self.assertEquals(line.qty_to_receive, 3,
                               "Wrong qty to receive")
             self.assertEquals(line.qty_incoming, 0,
                               "Wrong qty incoming")
             self.assertEquals(line.qty_to_deliver, 0,
                               "Wrong qty to deliver")
-            self.assertEquals(line.qty_outgoing, 0,
+            self.assertEquals(line.qty_outgoing, 3,
```

Comment:
If the picking is done (line 128) then the qty_outgoing should be zero but the quantity delivered 3, right?